src/platform/Linux/PlatformManagerImpl.cpp

```diff
@@ -170,13 +170,18 @@ void PlatformManagerImpl::WiFIIPChangeListener()
                     if (routeInfo->rta_type == IFA_LOCAL)
                     {
                         char name[IFNAMSIZ];
-                        ChipDeviceEvent event;
-                        if_indextoname(addressMessage->ifa_index, name);
+                        if (if_indextoname(addressMessage->ifa_index, name) == NULL)
+                        {
+                            ChipLogError(DeviceLayer, "Failed to get the interface name.");
```

Comment:
should we include the index into the message? so that debugging can look into what index failed and check for what could be the cause.

Also errno is useful here, to see if ENODEV or ENXIO or something like that.

Reply:
unresolving until the errno/index are added to the message (I believe the latest commit did not yet have the changes)

Reply:
Oh. I pushed the wrong version... thanks for double checking and sorry..

Reply:
Ok. That should be good now.